OS.md

```diff
@@ -0,0 +1,134 @@
+Operating Systeom (OS)
+===
+
+*Without an operating system your laptop is just a box of bits that do not know how to communicate with one another, or you !*
```

Comment:
The punctuation mark ALWAYS come after word without space and follow by one space.
```suggestion
*Without an operating system your laptop is just a box of bits that do not know how to communicate with one another, or you!*
```